app/components/search/search.component.ts

```diff
@@ -52,9 +52,8 @@ import { Component } from '@angular/core';
 export class SearchComponent {
     private value = '';
 
-    public onEnter(value: string) {
-        this.value = value;
-        console.log(value);
+    public onEnter() {
```

Comment:
You only need 1 function for both `keyup` and `click` events, since they are logically doing the same, but it's not critical now, just FYI.